newIDE/app/src/InstancesEditor/index.js

```diff
@@ -804,16 +811,69 @@ export default class InstancesEditor extends Component<Props> {
     this.viewPosition.scrollTo(x, y);
   }
 
-  centerView() {
+  zoomToFitContent() {
+    const instanceMeasurer = this.instancesRenderer.getInstanceMeasurer();
+    let rectangle: ?Rectangle;
```

Comment:
Maybe `contentRectangle` or `contentAABB`?

Reply:
I'll change it!